### PR TITLE
reloading commands with their alias

### PR DESCRIPTION
**Pull request by Clemens-E**
**Please describe the changes this PR makes and why it should be merged, as well as any and all proof of successful tests:**
allowing reloading of commands with their alias
credits to omer#0001 for locating and fixing the error before I pulled my fork
**Semantic versioning classification:**  
- [ ] This PR changes the framework's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to README, etc.

### CODE REVIEW

**Comment by AndrewJWin** [comment]
This merge request is well over a month old. Either fix the Linting issues or close the merge.

**Comment by Clemens-E** [comment]
> This merge request is well over a month old. Either fix the Linting issues or close the merge.

Sorry, forgot about this pr. :/

**Comment by AndrewJWin** [comment]
Proof
It does work, however I would recommend rewriting to tell the user _What_ command they reloaded.

I.e.

**-reload halp**
Would result with
 **\<ping>, The command ``help`` has been reloaded**

**Comment by AndrewJWin** [comment]
@YorkAARGH This PR request has been waiting merge since Febuary 12th. Please attend to this, either merge or close.

File: modules/functions.js [hunk 1]
```diff
@@ -108,13 +109,9 @@ module.exports = (client) => {
...
   client.unloadCommand = async (commandName) => {
-    let command;
-    if (client.commands.has(commandName)) {
-      command = client.commands.get(commandName);
-    } else if (client.aliases.has(commandName)) {
-      command = client.commands.get(client.aliases.get(commandName));
-    }
-    if (!command) return `The command \`${commandName}\` doesn"t seem to exist, nor is it an alias. Try again!`;
+    const command = client.commands.get(commandName) || client.commands.get(client.aliases.get(commandName));
+    commandName = command.help.name;
```
Comment (kyranet) [inline modules/functions.js]: btw if `command` is `undefined`, this will cause a TypeError.

**Review by kyranet — CHANGES_REQUESTED** [review]

**Comment by Clemens-E** [comment]
If no owner wants to review, merge, close, or even recognize this pr in any way, I can close it.

**Pull request closed without merging**